go.mod

```
@@ -0,0 +1,12 @@
+module github.com/reyhanfahlevi/workers
```

karena breaking changes, jadiin v2 aja

```suggestion
module github.com/reyhanfahlevi/workers/v2
```